src/main/webapp/app/shared/metis/post/post.component.ts

```
@@ -60,8 +60,10 @@ export class PostComponent extends PostingDirective<Post> implements OnInit, OnC
     readonly DisplayPriority = DisplayPriority;
 
     // Icons
-    faBullhorn = faBullhorn;
-    faCheckSquare = faCheckSquare;
+    readonly faBullhorn = faBullhorn;
+    readonly faPencilAlt = faPencilAlt;
+    readonly faTrash = faTrash;
+    @Input() isConsecutive: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)